www/data_module/README.md

```diff
@@ -25,3 +25,17 @@ $ gulp
 $ npm install
 $ karma start
 ```
+
+## How to test within buildbot/www/base ?
+
+```
+$ cp dist/* ../base/libs/buildbot-data/dist/
```

Comment:
I usually use symlink instead, so I can work both on the base and on the data service